zerver/lib/event_queue.py

```diff
@@ -343,7 +376,9 @@ def allocate_client_descriptor(new_queue_data):
     return client
 
 def do_gc_event_queues(to_remove, affected_users, affected_realms):
+    # type: (Set[str], Set[int], Set[int]) -> None
```

Comment:
Replace `Set` by [`AbstractSet`](https://docs.python.org/3/library/typing.html#typing.AbstractSet) here. It's usually better to use abstract base types as type of parameters (see [cheat sheet](https://github.com/python/mypy/blob/master/docs/source/cheat_sheet.rst#standard-duck-types)).